src/GitVersionTfsTask/Tasks/GitVersion/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "git-version",
+  "version": "1.0.0",
+  "private": true,
```

Comment:
this is not meant to be published to npm, I allows some of the usually fields required by package.json to be missing.